packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -12,6 +12,8 @@ import {scope} from '../../../reporters/detailed-collector';
 import {GeneratedAnswerActions as Actions} from './generated-answer-actions';
 import {GeneratedAnswerExpectations as Expect} from './generated-answer-expectations';
 
+const GENERATED_ANSWER_VISIBILITY_KEY = 'generatedAnswerIsVisible';
```

Comment:
In Atomic, local storage keys are formatted like this: `coveo-{component name}-data`. I'd suggest to use the same here. Also, in Atomic, I opted to store the visibility value in a data object so we can add additional information to the same storage if needed. I'd suggest to align the components to work the same way.

Reply:
Very good point! I will update accordingly